src/api-reference/financial-integration/v4.financial-documents-schema.markdown

```diff
@@ -139,6 +139,249 @@ Name|Type|Format|Description
 `cashAdvanceId`|`String`|-|GUUID. Unique system ID assigned to the cash advance. Maximum length 32 characters.
 `cashAdvanceTransactionType`|`Number`|-|Type of transaction: 1 = Issue or return to administrator, 2 = Application, including cash advance return expense within a report, or 3 = System cash advance, from balance carry forward.
 
+## Payroll <a name="scpayroll"></a>
+Name|Type| Format                                                        |Description
+---|---|---------------------------------------------------------------|---
+`report`|`Object`| [Payroll - Report](#scprlrep)                                 |***
+`employee`|`Object`| [Payroll - Employee](#scexpemploy)                            |***
+`cashAdvanceApplication`|`Object`| [Payroll - CashAdvanceUtilizationData](#scprlcaa)             |Currency. Cash advance utilized amount on a given expense report
+`expenseEntry`|`Array`| Array of [Payroll - ExpenseEntry](#scprlexpent)               |***
```

Comment:
This should link to the "Payroll - Employee" schema but instead links to the "Expense - Employee" schema. Please fix this.

Reply:
Fixed. Tested and it works.